lib/home/profile/account/account_settings.dart

```diff
@@ -48,25 +48,8 @@ class _AccountSettingsState extends State<AccountSettings> {
             1 => ListTile(
                 leading: const Icon(Icons.lock_outline),
                 title: const Text('change password'),
-                onTap: () => navigator.showDialog(
-                  AlertDialog.adaptive(
-                    title: const Text('Change Password'),
-                    content: const Text(
-                      'Are you sure you want to change the password?\n'
-                      "You'll need to enter your current password & new password to change.",
-                    ),
-                    actions: [
-                      ElevatedButton(onPressed: navigator.pop, child: const Text('No')),
-                      ElevatedButton(
-                        onPressed: () => navigator
-                          ..pop()
-                          ..push(const ChangePasswordScreen()),
-                        child: const Text('Yes'),
-                      ),
-                    ],
-                    actionsAlignment: MainAxisAlignment.spaceEvenly,
-                  ),
-                ),
+                onTap: () => Navigator.of(context).push(MaterialPageRoute(
+                    builder: (_) => const ChangePasswordScreen())),
```

Comment:
We can use the [`navigator`](https://github.com/theheartcenter-inc/thc/blob/e24a917a5bd1a4bc04c27847a1f85fc40e54f0aa/lib/utils/navigator.dart#L25) to make this even more concise

```suggestion
                onTap: () => navigator.push(const ChangePasswordScreen()),
```